src/gluonts/nursery/duru/data.py

```diff
@@ -0,0 +1,488 @@
+# Copyright 2018 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License").
+# You may not use this file except in compliance with the License.
+# A copy of the License is located at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# or in the "license" file accompanying this file. This file is distributed
+# on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+# express or implied. See the License for the specific language governing
+# permissions and limitations under the License.
+import gluonts.dataset
```

Comment:
Can you group the imports by

* Standard library imports.
* Related third party imports.
* Local application/library specific imports.

See: https://peps.python.org/pep-0008/#imports

Reply:
Done.